readthedocs/integrations/models.py

```diff
@@ -305,7 +305,7 @@ def save(self, *args, **kwargs):
             if token is None:
                 token = default_token()
                 self.provider_data = {'token': token}
-        super(GenericAPIWebhook, self).save(*args, **kwargs)
```

Comment:
Is there a reason we're removing *args? If someone passes in args, we don't want it to explode?